fastcrawler_ui/routers/sse.py

```diff
@@ -0,0 +1,135 @@
+from fastapi.responses import StreamingResponse
+from fastapi import APIRouter
+
+from uuid import uuid4
+import datetime
+import asyncio
+import random
+import json
+
+
+async def generate_random_log():
```

Comment:
if this is intended to be used as "mock" then create another package, and put it there. 
also speficy this with like #TODO: remove mock later